lib/active_model/serializer/configuration.rb

```diff
@@ -19,6 +19,7 @@ def config.array_serializer
           collection_serializer
         end
 
+        config.default_include_tree = '*'
```

Comment:
I've pushed up an updated patch that shows what this looks like with `default_includes`, which feels like a fairly reasonable name to set the default value used for the `include` option.

I don't really love any of the options we've talked about so far though, so I thought that addressing the awkward naming with documentation was probably a better approach than bikeshedding the name too much.

If someone has strong opinions, I'm totally open to changing the name and submitting again.